src/definitions/modules/modal.js

```diff
@@ -35,13 +35,7 @@
             methodInvoked  = typeof query === 'string',
             queryArguments = [].slice.call(arguments, 1),
 
-            requestAnimationFrame = window.requestAnimationFrame
-                || window.mozRequestAnimationFrame
-                || window.webkitRequestAnimationFrame
-                || window.msRequestAnimationFrame
-                || function (callback) {
-                    setTimeout(callback, 0);
-                },
+            requestAnimationFrame = window.requestAnimationFrame,
```

Comment:
can be used directly

Reply:
Yes, sure. I removed the declaration entirely now by e4e0f8226aa78e3ae185cebe7c4a09b6f4e223f6

Reply:
but access still needs to be done using `window.`, otherwise more eslint undef errors

Reply:
but the linter CI does not trigger an error?

Reply:
eslintrc.js already got
```
    env: {
        browser: true,
``` so eslint knows about the global functions

Reply:
I do not think so, the under rule is disabled.